testing/path/test_local.py

```diff
@@ -446,13 +453,6 @@ def notimpl(x, y):
         assert x.relto(tmpdir)
         assert x.check()
 
-    def test_make_numbered_dir_multiprocess_safe(self, tmpdir):
```

Comment:
The reason why this test was not catching the regression in the precious locking code for Windows is that it was being skipped on Windows (this class has the `pytestmark = skiponwin32` mark).